schedule/functional/skip_registration+workaround_modules.yaml

```diff
@@ -0,0 +1,58 @@
+name:           skip_registration+workaround_modules
+description:    >
+    Maintainer: slindomansilla
+    Like skip_registration test suite, but enable all modules as custom
+    addons. For internal testing, not simulating customer workflows.
+    See https://progress.opensuse.org/issues/25264 for details.
+    For now only "INSTALLONLY=1", later could be extended to do a full
+    test run.
+conditional_schedule:
+  bootloader:
+    BACKEND:
+      'qemu':
+        - installation/bootloader_start
+      'svirt':
+        - installation/bootloader_start
+      's390x':
+        - installation/bootloader_s390
```

Comment:
Did you try to make it work with `installation/bootloader_start`? Because that was the purpose of the module to unify schedules and create abstraction level which handles which modules to schedule. If it doesn't, please, let us know, as it should be easy to fix.

Reply:
Yes, and it didn't: https://openqa.suse.de/tests/4405898#step/bootloader_start/50

I didn't look deeper, I just wanted the YAML to get there ASAP.

Reply:
At least on the same machine type it works: https://openqa.suse.de/tests/4345185 I wonder that failure actually related to https://progress.opensuse.org/issues/33127 I will take a closer look to see if there is anything obvious.

Reply:
So I strongly believe that it should work, as under the hood with the variables you have in that run we would call `bootloader_s390` so it's equivalent. But of course that can be addressed later on and should not prevent you from using this schedule.

Reply:
seems not related, s390x-zVM-vswitch-l3 is working fine but s390x-zVM-vswitch-l2 seems to be broken, I tried with this job failing https://openqa.suse.de/tests/4428203 and succeeds changing MACHINE and S390X_NETWORK_PARAM (which are correlated) https://openqa.suse.de/tests/4428211

Reply:
This is now fixed, so it's not clear to me what went wrong.

Reply:
Just a quick update, issues on zVM are solved now, but you need to start test with `isos post` to get latest machine parameters. Other than that I strongly believe that `bootloader_start` module will work which will simplify this schedule.

Reply:
@rwx788, https://openqa.suse.de/tests/4484277#step/bootloader_start/36